modules/ROOT/pages/dataweave-flow-control-precedence.adoc

```diff
@@ -97,6 +99,76 @@ a|
 * `if else`: Conditional expression used for flow control in DataWeave.
 |===
 
+== Order of Chained Function Calls
+
+When functions are chained together in a series, the functions are processed
+in order from the first to the last function specified in the chain.
```

Comment:
This is a silly little thing, but I think it's redundant to say "in order":
```suggestion
from the first to the last function specified in the chain.
```

Reply:
yeah, I'm really want to emphasize it here, I think, since it's so important. Maybe overkill...